src/glossary.js

```diff
@@ -89,21 +93,48 @@ function Glossary(terms, selectors) {
   this.addEventListener(document.body, 'keyup', this.handleKeyup.bind(this));
 }
 
-/** Populate internal list.js list of terms */
 Glossary.prototype.populate = function() {
+  var self = this;
+  var terms = this.terms.map(function(term) {
+    return {
+      'term': term.term,
+      'definition': term.definition,
+      'termClass': self.classes.termClass,
+      'definitionClass': self.classes.definitionClass
+    };
+  });
+
+  var itemTemplate = _.template(
+    '<li>' +
+      '<button class="data-glossary-term {{ termClass }}">{{ term }}' +
+      '</button>' +
+      '<div class="{{ definitionClass }}">{{ definition }}</div>' +
+    '</li>',
+    {interpolate: /\{\{(.+?)\}\}/g}
+  );
+
+  terms.forEach(function(term) {
+    this.list.insertAdjacentHTML('beforeend', itemTemplate(term));
+  }, this);
+};
+
+/** Initialize list.js list of terms */
+Glossary.prototype.initList = function() {
+  var glossaryId = this.selectors.body.slice(1);
```

Comment:
Looks like we're still `slice`-ing the `selectors.body` selector and assuming it's going to be an ID. I continue to think that we shouldn't call the ID a selector--since it *has* to be an ID, let's call it `glossaryID`, or something similar. Looks like `list.js` simply calls this parameter `id` (although it actually accepts an ID or a DOM element, meaning our code does the same).